### ISSUE TITLE
Error: The given swagger file could not be found

### ISSUE THREAD

**Issue by dantesun** [vajahath/open-swagger-ui#8]
```
❯ open-swagger-ui ./open-api-v2.yaml --open
Error: The given swagger file (/home/dsun/workspace/.../open-api-v2.yaml) could not be found.
    at /usr/local/lib/node_modules/open-swagger-ui/dist/app.js:126:27
    at step (/usr/local/lib/node_modules/open-swagger-ui/dist/app.js:33:23)
    at Object.throw (/usr/local/lib/node_modules/open-swagger-ui/dist/app.js:14:53)
    at rejected (/usr/local/lib/node_modules/open-swagger-ui/dist/app.js:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
❯ open-swagger-ui -V
1.0.1
❯ node -v
v14.15.5
❯ npm -v
7.6.3
```

**Comment by vajahath** [comment]
Related #7.
@dantesun Could you please look into this and investigate this issue?
Else, I'll be checking this issue end of March as I'm a bit tight with my schedules 🙇

**Comment by vajahath** [comment]
@dantesun Released [1.1.0](https://github.com/vajahath/open-swagger-ui/releases/tag/v1.1.0) to npm fixing this issue.
